docs/sections/user_guide/cli/drivers/upp/help.out

```diff
@@ -12,6 +12,8 @@ Optional arguments:
 
 Positional arguments:
   TASK
+    control_file
+      The GRIB control file
```

Comment:
This must have been missed in a previous update -- it is unrelated to the changes in this PR.